packages/design-system/src/components/OcSpinner/OcSpinner.vue

```diff
@@ -37,7 +37,6 @@ const { ariaLabel = '', size = 'medium' } = defineProps<Props>()
 <style lang="scss">
 .oc-spinner {
   &::after {
-    border-bottom-color: transparent;
     content: '';
```

Comment:
did you already set `after:content-[""]` in another PR? if not you could do that here and delete the `style` blog :sunglasses

Reply:
I did it in the other PR. Should've done this in one PR now in hindsight 😅